__tests__/app.test.js

```diff
@@ -0,0 +1,52 @@
+const app = require('../app/app.js')
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+const db = require('../db/connection')
+beforeEach(() => 
+{
+    return seed(testData)
+})
+
+afterAll(() =>
+{
+    db.end()
+})
+
+describe('APP be-nc-news', () =>
+{
+    describe('GET /api/topics', () => 
+    {
+        test('GET: will return status 200', () => 
+        {
+            return request(app).get('/api/topics').expect(200)
+        })
+        test('responds with an array of objects', () => 
+        {
+            return request(app).get('/api/topics').expect(200)
+            .then((response) => 
+            {
+                const { body } = response
+                expect(Array.isArray(body)).toBe(true)
+                body.forEach((element) =>
```

Comment:
Make sure you check for length on the array. If your array comes back empty, this test will currently give you a false positive.